ci/webhook-receiver.py

```diff
@@ -166,6 +167,15 @@ def run_manager(request_ctx):
     log_access_granted(request_ctx, "Running CI")
     os.system("./manager.sh config {} \"{}\" \"{}\"".format(request_ctx['id'], request_ctx['repo'], request_ctx['body']))
 
+def poll_request_list():
+    while True:
+        # continuously check for waiting requests
+        if len(ci_request_list) > 0:
+            run_manager(ci_request_list.pop(0))
+        else:
+            # no requests, sleep
+            time.sleep(1)
```

Comment:
Figured we could just sleep for a second, but not sure how long is best

Reply:
Might want a sleeping queue sort of implementation in the future, but good enough for now

Reply:
what do you mean?